### ISSUE TITLE
clouddataservices2 link redirecting

### ISSUE THREAD

**Issue by vabarbosa** [ibm-cds-labs/dW#113]
the following pages redirect to the old page:

https://developer.ibm.com/clouddataservices2/start-developing-with-spark-and-notebooks

https://developer.ibm.com/clouddataservices2/sentiment-analysis-of-twitter-hashtags

**Comment by jessmantaro** [comment]
These have been converted into blog posts and should redirect to those posts. Thought this was done. Perhaps it will work when we switch?  Raj? Here are the URLs:
- https://developer.ibm.com/clouddataservices2/sentiment-analysis-of-twitter-hashtags/ to: https://developer.ibm.com/clouddataservices2/2015/10/06/sentiment-analysis-of-twitter-hashtags/
- https://developer.ibm.com/clouddataservices2/start-developing-with-spark-and-notebooks/ to https://developer.ibm.com/clouddataservices2/2015/10/07/start-developing-with-spark-and-notebooks/

**Comment by isc-rsingh** [comment]
Yes the idea is that they will work properly when we switch. Adding a milestone for this.
